src/pages/workspaces/workspace/applications/AppLauncher.js

```diff
@@ -79,7 +79,7 @@ const AppLauncher = _.flow(
           namespace, name, clusters, persistentDisks,
           onDismiss: () => setShowCreate(false),
           onSuccess: _.flow(
-            withErrorReporting('Error creating cluster'),
+            withErrorReporting('Error creating runtime'),
```

Comment:
Nit: This should probably reference "cloud environment" instead of "runtime".
TOL: I think this error message exists elsewhere, too. It would be nice if somehow `NewClusterModal` could be responsible for this error message.

Reply:
Good call, and actually this should say 'Error loading cloud environment', since that's all it does now.